### ISSUE TITLE
[CSS] Update link and hover colours

### ISSUE THREAD

**Issue by sboots** [ytgov/ckanext-yukondesign#112]
**Estimated hours: 2**

~As it turns out I used an older version of our Yukon style colours in the sample SCSS Bootstrap template.~ 😭 Update: turns out the SCSS Bootstrap template [uses the correct colours](https://github.com/ytgov/yukon-bootstrap5-sass-js/blob/main/scss/styles.scss#L55-L60). Rather, it was the earlier Figma designs that used out-of-date ones. 😜 

The intended colours are:

```
Link: #005a65
Hover: #008392
```

In the SCSS Bootstrap repository, they're [implemented as follows](https://github.com/ytgov/yukon-bootstrap5-sass-js/blob/main/scss/styles.scss#L55-L60):

```
// Link colours

$link-color: #005a65;
$link-hover-color: #008392;

$primary: #005a65;
```

~I'll update the Bootstrap template as soon as I have a chance; I think I had these the other way around.~ 😅  

Please update the link color and link hover colours, and primary button colours, to match the hex codes above. This should also apply to primary button colours (anything using the Bootstrap "primary" template colour, essentially). Thanks so much!

**Comment by sboots** [comment]
Updated the description above - the GitHub SCSS Bootstrap repository is correct, as it turns out (the earlier Figma designs didn't use the updated colours).
https://open-data-static-files.service.yukon.ca/yukon-bootstrap5-sass-js/
https://github.com/ytgov/yukon-bootstrap5-sass-js/blob/main/scss/styles.scss#L55-L60

Thanks!

**Comment by willy1989cv** [comment]
So can we close this @sboots ?

**Comment by sboots** [comment]
On the dev site, the links are still showing as the slightly lighter colour. It looks like it's still using,
```
 --bs-link-color-rgb: 0, 131, 146;
```
...which is the slightly lighter colour from the Figma designs, and not the intended `#005a65` link colour. 

Instead, that should be (using the RGB equivalent of #005a65)
```
 --bs-link-color-rgb: 0, 90, 101;
```

and the hover colour should be,
```
--bs-link-hover-color: #008392;
```

I think, at least. 😅 Thanks!!

**Comment by willy1989cv** [comment]
@sboots please validate again. this was implemented on the "add link underline issue" which was merged today

**Comment by sboots** [comment]
Awesome, thanks @willy1989cv ! Looks great! 

The only charge I'd like is, 

- in the footer, the "Government of Yukon" and "Submit website feedback" links should stay white when hovered over, rather than turning blue like they currently do. 

Thanks a ton for updating these!

**Comment by gavram** [comment]
@sboots we spent a bit more time on improving this, also on testing and deployment. Our estimate is 4 hours
